packages/ui/src/AdminMCQ.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+import { useEffect, useState } from "react";
+import { createMCQ, getAllMCQs } from "../../../apps/web/components/utils";
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "./shad/ui/card";
+import { Button } from "./shad/ui/button";
+import { Problem, MCQQuestion } from "@prisma/client";
+import AdminAddMCQ from "./AdminAddMCQ";
+
+interface AdminAddMCQProps extends Problem {
+  mcqQuestions: MCQQuestion[];
+}
+
+const AdminMCQ = () => {
+  const [mcq, setMcq] = useState<MCQQuestion>();
+  const [Problems, setProblems] = useState<AdminAddMCQProps[]>([]);
+  useEffect(() => {
+    async function fetchMCQs() {
+      const Problems = await getAllMCQs();
```

Comment:
Use try catch block for this async function

Reply:
https://github.com/code100x/daily-code/pull/311#discussion_r1573262251